### PR TITLE
Add proxy support

### PR DESCRIPTION
**Pull request by phant0m0day**
Add proxy support for web requests made during brute-force. Allows users to choose or not choose a proxy. Hopefully better implementation later, but for now, a working proxy brute-force. Working on adding changing proxy every time with a proxy list

### CODE REVIEW

File: src/facebookLib.py [hunk 1]
```diff
@@ -25,15 +27,27 @@ def facebookCheck(username):
 
 
 def facebookBruteforce(username, wordlist, delay):
-    driver = webdriver.Firefox()
```
Comment (phant0m0day) [inline src/facebookLib.py]: Simple recommendation: make proxy a command line argument for easy use.

File: src/facebookLib.py [hunk 2]
```diff
@@ -44,6 +58,6 @@ def facebookBruteforce(username, wordlist, delay):
         except AssertionError:
             print G + "[*] Username: %s | [*] Password found: %s\n" % (username, password) + W
             sys.exit(0)
-        except Exception, e :
+        except Exception, e:
```
Comment (phant0m0day) [inline src/facebookLib.py]: Nothing here, just removed whitespace between 'e' and ':' :+1:

**Comment by ex0dus-0x** [comment]
Thanks for all the changes. However, I won't merge the pull request, as I want to write uniform code. This is very effective and I will definitely implement this.

**Comment by R3C0Nx00** [comment]
Understandable, it was just a concept. Maybe use a proxy list API for random HTTP/HTTPs proxies. @ex0dus-0x

**Pull request merged**